src/metaswitch/clearwater/etcd_tests/etcdserver.py

```diff
@@ -8,7 +8,7 @@
 from shutil import rmtree
 import uuid
 
-base_cmd =              """clearwater-etcd/usr/bin/etcd --debug --listen-client-urls http://{0}:4000 --advertise-client-urls http://{0}:4000 --listen-peer-urls http://{0}:2380 --initial-advertise-peer-urls http://{0}:2380 --data-dir {2} --name {1}"""
+base_cmd =              """clearwater-etcd/usr/share/clearwater/clearwater-etcd/3.1.7/etcd --debug --listen-client-urls http://{0}:4000 --advertise-client-urls http://{0}:4000 --listen-peer-urls http://{0}:2380 --initial-advertise-peer-urls http://{0}:2380 --data-dir {2} --name {1}"""
```

Comment:
This means all the tests now run against 3.1.7 and not 2.2.5 right? Is it ok to stop all testing of 2.2.5 when customers will need to use it during the acceptance phase. What if we write some code that only works on 3.1.7 and not 2.2.5?

Is there a way to test both or at least have the option to test on 2.2.5?